src/Components/Navbar/Navbar.js

```diff
@@ -0,0 +1,58 @@
+function Navbar() {
+  // in this component I am using "require" inside the src of the img to import images dynamiclly
+  const NavArr = [
```

Comment:
I would name this something like `navElements` starting with small case letter or `NAV_ELEMENTS` because its all constant and maybe put it in a constants folder

Reply:
constants folder?

Reply:
yes you can create that in the `src` folder directly